modules/services/redshift-gammastep/lib/options.nix

```
@@ -110,6 +110,8 @@ in {
       '';
     };
 
+    verbose = mkEnableOption "verbose service logging" // { default = true; };
```

I'm not sure why verbose logging is enabled by default, it's mostly interesting for debug purposes and could be off most of the time. So perhaps simply have it default to false?

I would prefer to rename the option to something like `enableVerboseLogging` or something similarly … verbose.

So in summary:

```suggestion
    enableVerboseLogging = mkEnableOption "verbose service logging";
```

---

Yeah I only left it on for backward-compatibility with existing configs, but I personally agree that it makes more sense to be off by default. Done!